monkey/monkey_island/cc/ui/src/services/configuration/masquerade.js

```diff
@@ -0,0 +1,34 @@
+const MASQUERADE = {
+  'title': 'Masquerade',
+  'type': 'object',
+  'description': 'To identify malware in a file, one common approach is to search for specific ' +
+  'strings or unique byte sequences.\nIn order to enhance Infection Monkey\'s malware emulation,'+
+  'custom data can be injected into agent binaries, allowing the agents to imitate various types of malware. '+
+  'This feature can be particularly helpful when testing custom detection rules.',
+  'properties': {
+    'linux_masque_strings': {
+      'title': 'Linux Masque Strings',
+      'type': 'array',
+      'uniqueItems': true,
+      'items': {'type': 'string'},
+      'default': [],
+      'description': 'List of masque strings that will be insert to the Linux Agent binaries '
```

Comment:
```suggestion
      'description': 'List of masque strings that will be applied to the Linux Agent binary'
```
We've used "apply" everywhere in the backend. Let's stick to one.
Same for Windows.